### ISSUE TITLE
Incorrect redundant default suggestion

### ISSUE THREAD

**Issue by Dynom** [golang/lint#281]
The following produces this warning:
```
warning: should drop = "" from declaration of var a; it is the zero value (golint)
```

The definition:
```go
var a interface{} = ""
```
However the default value of `interface{}` is `nil`, so the declaration is not redundant.

A work around:
```go
var a = interface{}("")
```
